app/models/immunisation_import_row.rb

```diff
@@ -402,4 +396,18 @@ def zero_or_one_existing_patient
       errors.add(:patient, :multiple_duplicate_match)
     end
   end
+
+  def patient_attributes
+    {
+      address_postcode: patient_postcode,
+      date_of_birth: patient_date_of_birth,
+      first_name: patient_first_name,
+      gender_code: patient_gender_code,
+      home_educated:,
+      imported_from:,
+      last_name: patient_last_name,
+      nhs_number: patient_nhs_number,
+      school:
```

Comment:
How would we handle `school` (and `imported_from`) since these are foreign keys? Do we need to add something to `stage_changes` to store the ID in the `pending_changes` field?

Reply:
Good point. Maybe we should silently omit these keys from the `stage_changes` call?

Reply:
I agree about `imported_from`, but I think for `school` it actually works quite nicely if we could support it as part of this feature, if a patient has moved schools between imports then it would be good to flag it like the other changed fields.

Reply:
@thomasleese yeah, fair point, let me update it to work with schools and add a test 👍